### DISCUSSION TITLE
The number of edges in datasets  “Cora”, “CiteSeer” and “PubMed” is inconsistent with the paper description

### DISCUSSION THREAD

**Discussion by xiaomi4356** [pyg-team/pytorch_geometric#6203] [Q&A] [answered]
The number of datasets “Cora”, “CiteSeer” and “PubMed” edges is inconsistent with the paper description
in class Planetoid：
Cora has 10556 edges, but in the paper 'Revisiting Semi-Supervised Learning with Graph Embeddings' , Cora has 5429 edges, 10556/2=5278, not 5429, why?
the same doubt as datasets “CiteSeer” and “PubMed”

**Comment by xiaomi4356** [comment]
I checked the description of the database for other papers，the Cora dataset has 5429 edges，Why is the number of sides different in Planetoid?

**Comment by EdisonLeeeee** [comment, marked as answer]
Taking Cora as an example, there are edges that are inherently undirected, e.g., (1,2) and (2, 1) both exist in the dataset. PyG will remove duplicate edges from a graph when converting it to an undirected one, so the number of edges will be less than 5429 * 2.
